### PR TITLE
Issue 1347 use screensize

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

File: Packages/vcs/Lib/VTKPlots.py [hunk 1]
```diff
@@ -406,9 +406,11 @@ def portrait(self, W, H, x, y, clear):
             self.renWin.SetSize(W, H)
 
     def initialSize(self):
-        # screenSize = self.renWin.GetScreenSize()
-        self.renWin.SetSize(self.canvas.bgX, self.canvas.bgY)
-        self._lastSize = (self.canvas.bgX, self.canvas.bgY)
+        screenSize = self.renWin.GetScreenSize()
+        bgX = int(screenSize[0]*.6)
```
Comment (aashish24) [inline Packages/vcs/Lib/VTKPlots.py]: what are these magic numbers? (0.6). It would be nice if you can add a comment here.

Comment (doutriaux1) [inline Packages/vcs/Lib/VTKPlots.py]: 60% of the screen size, I thought it was obvious... That's a reasonable size but if you come up with a percentage you prefer feel free to update :wink:

**Comment by doutriaux1** [comment]
@aashish24 ready to be merged in

**Comment by aashish24** [comment]
LGTM :+1:

**Pull request merged**